### DISCUSSION TITLE
执行失败的sql和执行成功的sql默认日志级别都是info，有没有办法可以区别开不同的日志级别

### DISCUSSION THREAD

**Discussion by p-mega** [rbatis/rbatis#548] [Q&A]
例如：  
  #[py_sql("an error sql")]
    pub async fn err_sql_exec(rb: &dyn Executor) -> Result<Vec<Self>, Error> {
        impled!()
    }
这条sql执行失败，但仍然打印一条info级别的日志

**Comment by zhuxiujia** [comment]
> 例如： #[py_sql("an error sql")] pub async fn err_sql_exec(rb: &dyn Executor) -> Result<Vec, Error> { impled!() } 这条sql执行失败，但仍然打印一条info级别的日志

目前sql和错误都是info打印的，如果你需要分级打印的话，可以自定义一个日志Interceptor拦截器，并把默认的日志拦截器移除
